apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1357,7 +1358,11 @@ namespace MWPhysics
         mMovementResults.clear();
 
         mTimeAccum += dt;
-        const float physicsDt = 1.f/60.0f;
+
+        static const float physFramerate = Settings::Manager::getFloat("physics framerate", "Physics");
```

Comment:
You should avoid local ```static const``` variables that are initialized like this in time-sensitive code. They aren't actually initialized until the first time the code is reached and thus needs hidden atomic checks, branches, and locks to ensure it's initialized only once and are safely reentrant (the compiler can't know if multiple threads may hit that code at the same time).

Reply:
Is it really that time sensitive code though? This function is only called once per frame.